app/src/main/java/org/mozilla/fenix/settings/address/AddressEditorFragment.kt

```diff
@@ -26,6 +30,15 @@ class AddressEditorFragment : SecureFragment(R.layout.fragment_address_editor) {
 
     private lateinit var addressEditorView: AddressEditorView
     private lateinit var interactor: AddressEditorInteractor
+    private val args by navArgs<AddressEditorFragmentArgs>()
+
+    /**
+     * Returns true if an existing address is being edited, and false otherwise.
+     */
+    private val isEditing: Boolean
+        get() = args.address != null
+
+    private lateinit var menu: Menu
```

Comment:
I don' think we actually needed this `menu` variable. Even in `CreditCardEditorFragment` I didn't think we needed it because it was done so that we can call `menu.close()` in `onPause()`, but the menu never really opens, and I think we just ended up copying code from logins without fully understanding it.

Reply:
The menu opens if the toolbar text is long enough and the device screen is short. I think it will be good if we keep the same pattern everywhere. Also, we could add some tests for it if we keep it.

Reply:
That makes sense. Should we add the `menu.close()` handling in `onPause()` in this PR as well? Otherwise, we just have this `menu` that isn't doing anything currently.